db/migrate/20220726073904_add_user_ref_to__food.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToFood < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :foods, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
Food does not have a relationship with a user. I suggest you remove the `user_id` from foods table